src/main/java/org/truffleruby/core/cast/ToStrNode.java

```diff
@@ -29,6 +29,10 @@
 
     public abstract DynamicObject executeToStr(VirtualFrame frame, Object object);
 
+    public static ToStrNode create() {
+        return ToStrNodeGen.create(null);
+    }
```

Comment:
I wish the DSL did this automatically as well.

Reply:
True. In Graal.js we manually wrote a create method for most Nodes. DSL could at least understand the "(null)" pattern.